lightautoml/automl/base.py

```diff
@@ -118,6 +120,7 @@ def _initialize(
                 input features to next levels.
             return_all_predictions: True if we should return all predictions from last
                 level models.
+            debug: To catch running model exceptions or not.
```

Comment:
add more information about behaviour, for example:
True: show exceptions during model training;
False: catch and hide exceptions;